HotelManagement/src/core/Booking.java

```diff
@@ -0,0 +1,80 @@
+package core;
+
+import java.time.LocalDate;
+
+public class Booking {
+	private LocalDate fromDate;
+	private LocalDate toDate;
+	private String guestName;
+	private String guestId;
+
+	public Booking(LocalDate fromDate, LocalDate toDate, String guestName, String guestId) {
+		if (fromDate.isBefore(toDate)) {
```

Comment:
Since you're doing the date validation check in the constructor, you should also check if the parameters are not null. If you get a null fromDate the first if will throw an exception.

Reply:
Done